.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,44 @@
+---
+name: Bug report
+about: Create a report to help us improve
+title: ''
+labels: bug
+assignees: ''
+
+---
+
+<!--
+Thank you for reporting a possible bug in nodejs-itoolkit.
+
+Please fill in as much of the template below as you can.
+
+Node.js version: output of `node -v`
+itoolkit version: output of `npm ls itoolkit`
+IBM i version: output of `uname -vr`
```

Comment:
Should add XMLSERVICE version: `qsh cmd('/qsys.lib/qxmlserv.lib/xmlver.pgm')`

Reply:
Added